README.md

```diff
@@ -2652,7 +2652,7 @@ Unit testing helps maintain clean code, as such I included some of my recommenda
           /**
            * @name logError
            * @desc Logs errors
-           * @param {String} msg Message to log
+           * @param {String} msg - Message to log.
            * @returns {String}
```

Comment:
Changed it. 

This is what is officially in JSDoc documentation. Will welcome any other opinions on this.